src/templates/ApiReference.js

```diff
@@ -120,7 +120,7 @@ const NavItemEl = styled.div`
 
 const StyledLink = components.a;
 
-const NavLinkEl = styled(BasicLink)`
+const NavLinkEl = styled(Link)`
```

Comment:
Turns out this is why links on api ref were slow! Was doing a regular gastby route transition, totally ignoring the optimizations we'd out in place

Reply:
Nice!!